tasks/main.yml

```diff
@@ -32,26 +32,26 @@
     mode: '0755'
   when: dell_dsu_rollout_gpg_keys
 
-- name: copy gpg keys
-  copy:
-    dest: /usr/libexec/dell_dup/{{ item }}
-    src: "{{ item }}"
-  loop: "{{ dell_dsu_gpg_keys }}"
-  when: dell_dsu_rollout_gpg_keys
+# - name: copy gpg keys
```

Comment:
These could be removed instead of commented.

Reply:
fixed